packages/test/stochastic-test-utils/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "@fluid-internal/stochastic-test-utils",
```

Comment:
Can you make sure that the CODEOWNERS entry that covers this project is correct?

Reply:
The `/package/test/` pointing to `@microsoft/fluid-cr-test` should cover this, right?

Reply:
if not, can handle this in a follow-up. Completing now.